### DISCUSSION TITLE
Other OAUTH methods?

### DISCUSSION THREAD

**Discussion by ghost** [dyc3/opentogethertube#1143] [Q&A] [answered]
Does OTT support OAUTH methods other than Discord?

I tried digging through the repo but couldn't find much.

**Comment by dyc3** [comment, marked as answer]
Nope, not currently. But I would be open to adding more.

**Reply by ghost** [reply]
Would certainly be lovely if I could use my self-hosted SSO (like Authelia, Keyclock, etc.) rather than relying on a random 3rd party...or just openly allowing signups. 😅

**Reply by dyc3** [reply]
I gave those a quick glance, and I don't think authelia would fit very well in OTT's current architecture. Maybe keycloak would.

I can definitely add a config option to disable signups. I opened #1145 for it.
